airtable/params.py

```diff
@@ -1,368 +1,462 @@
+
+        Limits the maximum number of records returned in each request.
+        Default is 100.
+
+        Usage:
+
+        >>> airtable.get_all(page_size=50)
+
+        Args:
+            page_size (``int``): The number of records returned in each request.
+                Must be less than or equal to 100. Default is 100.
+
+        """
+
+        # Class Input > Output
+        # >>> filter = PageSizeParam(50)
+        # >>> filter.to_param_dict()
+        # {'pageSize: 50}
+
+        param_name = "pageSize"
+        kwarg = "page_size"
+
+    class CellFormatParam(_BaseParam):
+        """
+        cellFormat Param
+
+        Kwargs:
+            ``cellFormat=``
+
+        If set, the format that should be used for cell values. Supported values are:
+           json: cells will be formatted as JSON, depending on the field type.
+           string: cells will be formatted as user-facing strings, regardless of the field type.
+                   Note: You should not rely on the format of these strings, as it is subject to change.
+
+        The default is json.
+
+        Usage:
+
+        >>> airtable.get_all(cellFormat='json')
+
+        Args:
+            cellFormat (``str``): The format to use.
+
+        """
+
+        # Class Input > Output
+        # >>> filter = CellFormatParam('The format to use.')
+        # >>> filter.to_param_dict()
+        # {'cellFormat: 'The format to use.'}
+
+        param_name = "cellFormat"
+        kwarg = param_name
+
+    class TimeZoneParam(_BaseParam):
+        """
+        timeZone Param
+
+        Kwargs:
+            ``timeZone=``
+
+        The time zone that should be used to format dates when using string as the cellFormat.
+        This parameter is required when using string as the cellFormat.
+        List of accepted values:
+        https://support.airtable.com/hc/en-us/articles/216141558-Supported-timezones-for-SET-TIMEZONE
+
+        No default. It is MANDATORY WHEN USING string as CellFormat.
+
+        Usage:
+
+        >>> airtable.get_all(timeZone='Europe/London')
+
+        Args:
+            timeZone (``str``): One of https://support.airtable.com/hc/en-us/articles/216141558-Supported-timezones-for-SET-TIMEZONE
+
+        """
+
+        # Class Input > Output
+        # >>> filter = TimeZoneParam('The timeZone to use.')
+        # >>> filter.to_param_dict()
+        # {'timeZone: 'The timeZone to use.'}
+
+        param_name = "timeZone"
+        kwarg = param_name
+
+    class UserLocaleParam(_BaseParam):
+        """
+        userLocale Param
+
+        Kwargs:
+            ``userLocale=``
+
+        The user locale that should be used to format dates when using string as the cellFormat.
+        This parameter is required when using string as the cellFormat.
+        List of accepted values:
+        https://support.airtable.com/hc/en-us/articles/220340268-Supported-locale-modifiers-for-SET-LOCALE
+
+        No default. It is MANDATORY WHEN USING string as CellFormat.
```

Comment:
does the api have clear msg when these are missing?
If so, maybe we should validate?